scapy/config.py

```diff
@@ -765,7 +767,7 @@ class Conf(ConfClass):
     min_pkt_size = 60
     #: holds MIB direct access dictionary
     mib = None  # type: 'scapy.asn1.mib.MIBDict'
-    bufsize = 2**16
+    bufsize = 2 ** 16
```

Comment:
Please avoid unneeded change to preserve the code history as much as possible.

Reply:
ok